### ISSUE TITLE
Installation failed...

### ISSUE THREAD

**Issue by Lxp1zero** [re1san/Kde-Dots#13]
![Screenshot_20240223_183725-1](https://github.com/re1san/K-Dots/assets/126608444/2471d823-6b3f-4333-8c28-b023fc53f6f6)

**Comment by re1san** [comment]
What OS are you using?

**Comment by Lxp1zero** [comment]
> What OS are you using?
Opensuse tw

**Comment by re1san** [comment]
Yeah I am missing some dependencies on opensuse so somethings will not not work like clock applet. (You can use Arch for everything to work)

**Comment by syn1de** [comment]
> Yeah I am missing some dependencies on opensuse so somethings will not work like clock applet. (You can use Arch for everything to work)

![2024-02-24_14-31](https://github.com/re1san/K-Dots/assets/141513673/3fc02dbd-6b26-45bd-b51e-c10af35b7f08)
 i am using arch linux still same error

**Comment by re1san** [comment]
@syn1de are you on fresh install?

**Comment by syn1de** [comment]
> @syn1de are you on fresh install?

yessir
